### PR TITLE
Update C* recovery process

### PR DESCRIPTION
**Pull request by feeblefakie**
https://scalar-labs.atlassian.net/browse/DLT-7200

### CODE REVIEW

File: docs/CassandraOperation.md [hunk 1]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
-
-Finally you can start the Cassandra service.
-
-```console
-sudo systemctl start cassandra
-```
-
-If you replaced a seed node (old seed node), you should replace the old seed node's IP of all Cassandra nodes with newly created Cassandra node IP.
+### Restart Cassandra
+As a last step of Cassandra node replacement, you need to configure Cassandra appropriately before start it. What needs to be done depends on the following cases.
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:123]:
```suggestion
As the last step of Cassandra node replacement, you need to configure Cassandra appropriately before starting it. What needs to be done depends on the following cases.
```

File: docs/CassandraOperation.md [hunk 2]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
+
+* Case1: Node (Instance/VM) is repalced (but the volume is not replaced)
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:125]:
```suggestion
* Case1: Node (Instance/VM) is replaced (but the volume is not replaced)
```

File: docs/CassandraOperation.md [hunk 3]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
+* Case2: Volume is repalced (but the node is not replaced)
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:126]:
```suggestion
* Case2: Volume is replaced (but the node is not replaced)
```

File: docs/CassandraOperation.md [hunk 4]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
+* Case3: Node and the volume are both replaced
+
+Here are the steps for each case.
+
+#### Case1: Node (Instance/VM) is repalced (but the volume is not replaced)
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:131]:
```suggestion
#### Case1: Node (Instance/VM) is replaced (but the volume is not replaced)
```

File: docs/CassandraOperation.md [hunk 5]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
+* Remove the IP of the new node from seeds in `casssandra.yaml`
+* Add `JVM_OPTS="$JVM_OPTS -Dcassandra.replace_address_first_boot=<dead node ip>"` to the bottom of `cassandra-env.sh`
+* `sudo systemctl start cassandra`
+* Repair with Reaper at some later point 
+
+#### Case2: Volume is repalced (but the node is not replaced)
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:137]:
```suggestion
#### Case2: Volume is replaced (but the node is not replaced)
```

File: docs/CassandraOperation.md [hunk 6]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
+* Restore data from Cassy
+* `sudo systemctl start cassandra`
+* Repair with Reaper at some later point 
+
+#### Case3: Node and the volume are both replaced
+* Restore data from Cassy
+  * You need to manually update the IP of an entry of `backup_history` table to match with the new IP
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:144]:
```suggestion
  * You need to manually update the IP of the entry of `backup_history` table to match with the new IP
```

**Review by ymorimo — DISMISSED** [review]
LGTM!
Added one minor comment.

File: docs/CassandraOperation.md [hunk 7]
```diff
@@ -119,25 +119,37 @@ sudo /bin/bash -c "echo 'UUID=af767839-b23d-4d1f-8a19-debbcfd6413c /data xfs def
...
+* Repair with Reaper at some later point 
+
+#### Case3: Node and the volume are both replaced
+* Restore data from Cassy
+  * You need to manually update the IP of the entry of `backup_history` table to match with the new IP
+* Remove the IP of the new node from seeds in `casssandra.yaml`
+* Set `auto_bootstrap` to `false` in `cassandra.yaml`
+    * this might not be needed. need to verify.
+* Add `JVM_OPTS="$JVM_OPTS -Dcassandra.replace_address_first_boot=<dead node ip>"` to the bottom of `cassandra-env.sh`
+* sudo systemctl start cassandra
```
Comment (feeblefakie) [inline docs/CassandraOperation.md:149]:
```suggestion
* `sudo systemctl start cassandra`
```

**Review by tei-k — APPROVED** [review]
LGTM!

**Review by yito88 — APPROVED** [review]
LGTM

**Pull request merged**